app/src/main/java/com/geeksville/mesh/navigation/NodesNavigation.kt

```diff
@@ -98,12 +114,6 @@ fun NavGraphBuilder.nodeDetailGraph(navController: NavHostController) {
                         entry,
                         entry.screenComposable,
                     )
-                is NodeDetailRoutes.NodeMap ->
-                    addNodeDetailScreenComposable<NodeDetailRoutes.NodeMap>(
-                        navController,
-                        entry,
-                        entry.screenComposable,
-                    )
```

Comment:
This enum pattern won't scale well with each screen having its own viewmodel.

Another advantage to breaking this up is each navigation route can have its own args which will allow use to get rid of the `parentGraphBackStackEntry` hack and enforce non-null args where we know they shouldn't be null (e.g. `destNum`).

Most of these screens have VM params with default values of `hiltViewModel()` which will never actually work because they expect nav args from the parent graph.